opus/application/static_media/css/opus.css

```diff
@@ -38,6 +38,15 @@ body {
     }
 }
 
+.op-update-url-msg {
```

Comment:
I think we want to make this entire message thing a little more generic - perhaps op-user-message or something along that line, so we can reuse if we choose to